README.md

```diff
@@ -148,10 +148,10 @@ sqlx = { version = "0.7", features = [ "runtime-async-std", "tls-rustls" ] }
 
 #### Cargo Feature Flags
 
-For backwards-compatibility reasons, the runtime and TLS features can either be chosen together as a single feature,
+For backward-compatibility reasons, the runtime and TLS features can either be chosen together as a single feature,
 or separately.
 
-For forward-compatibility, you should use the separate runtime and TLS features as the combination features may
+For forward compatibility, you should use the separate runtime and TLS features as the combination features may
```

Comment:
You deleted the hyphen in this line but kept it in "backward-compatibility" in the previous line.

Reply:
from https://www.grammarly.com/blog/hyphen/ 
`You also don’t need a hyphen when your modifier is made up of an adverb and an adjective.`

Reply:
Okay, I would rephrase the "backward-compatibility" line to match then, because this looks incongruous.